ckanext/data_qld/actions.py

```diff
@@ -91,6 +95,9 @@ def _undictize_datarequest_basic(data_request, data_dict):
     data_request.description = data_dict['description']
     organization = data_dict['organization_id']
     data_request.organization_id = organization if organization else None
+    if tk.h.closing_circumstances_enabled:
```

Comment:
please make a function as you have this duplicated twice here and on line 335.

Reply:
Refactored into function